sdk-core/src/main/java/care/data4life/sdk/network/util/interceptor/BasicAuthorizationInterceptor.kt

```diff
@@ -14,7 +14,7 @@
  * contact D4L by email to help@data4life.care.
  */
 
-package care.data4life.sdk.network.interceptor
+package care.data4life.sdk.network.util.interceptor
```

Comment:
why have they been moved to util?

Reply:
we could move them later

Reply:
I would keep interceptor

Reply:
Lets make a final decision once we reached #139.